comm_config.yaml

```
@@ -3,18 +3,24 @@ communications:
   # Settings for Slack
   slack:
     enabled: false
-    channel: 'SLACK_CHANNEL'
-    token: 'SLACK_API_TOKEN'
-    notiftype: short                           # Change notification type short/long you want to receive. notiftype is optional and Default notification type is short (if not specified)
+    token: 'SLACK_TOKEN'
+    notiftype: short                             # Change notification type short/long you want to receive. notiftype is optional and Default notification type is short (if not specified)
+    accessbindings:
```

Let's use camelcase syntax here
```suggestion
    accessBindings:
```